firecrown/likelihood/gauss_family/statistic/README.md

```diff
@@ -0,0 +1,73 @@
+# Documentation for the SACC format needed for number_counts_stats.py
+
```

Comment:
Naively looking at where this readme file is, it seems a bit strange to have documentation related specifically to cluster counts in what seems to be a very general folder, `firecrown/likelihood/gauss_family/statistic/`, that could be relevant for other probes?